### ISSUE TITLE
Need a reset button on GUI

### ISSUE THREAD

**Issue by radiolee** [LaserWeb/LaserWeb4#447]
I'm using STM32 port of grbl 1.1f now https://github.com/usbcnc/grbl/ it's a USB Virtual Com Port. I created an .env file with RESET_ON_CONNECT=1, and now can connect the board. But, after I generate the g-code, run the job, xyz still stay 0.00mm, Queued lines is decrease, but no move. I try to send ctrl-x; reset command but no luck. I touch the same issues in cncjs, but it's working ok after I click reset.

I can not do reset by push the button on the STM32 board, because it's reset usb vcp and loss connection when I push the hardware reset button. Laserweb will lost the comport and need connect again, the issue still again.

So, how can I send a soft reset(ctrl+x) and "?" to grbl by manual?

**Comment by tbfleming** [comment]
There's a keyboard shortcut. Press ctrl+x to send ctrl+x to the board.

**Comment by cprezzi** [comment]
Don't forget configure the grbl settings correctly (see https://cncpro.yurl.ch/documentation/initial-configuration/65-grbl-lpc-1-1e) and load the "Generic GRBL Machine" Profile in LW4.

**Comment by radiolee** [comment]
thanks for help, I'll check it  later.

**Comment by radiolee** [comment]
@cprezzi @tbfleming Thanks all.  It's working send ctrl+x, after setting grbl LW4 works normally. In the guide https://cncpro.yurl.ch/documentation/initial-configuration/65-grbl-lpc-1-1e I found this line:
$10=0      ;send work coordinates in statusReport (needed for LW4!)
is it mean LW4 only accecpt WPos report? I think this is the point of my issues when I leave $10=1, that's report MPos. But I found this https://github.com/LaserWeb/deprecated-LaserWeb3/pull/172 by @cprezzi in LW3 status report can fallback from WPos to MPos, so I was not care about it.
